cg/cli/workflow/microsalt/base.py

```diff
@@ -31,6 +31,7 @@ def microsalt(context: click.Context, ticket: str, dry_run: bool):
         hk_api=hk_api,
         lims_api=lims_api,
         fastq_handler=FastqHandler(context.obj),
+        config=context.obj,
```

Comment:
Suggest to only send the configs that are necessary, is it possible here to send:

```python
context.obj["microsalt"]
```
or something like that?

Reply:
that makes a lot of sense, changed it